src/App.tsx

```diff
@@ -17,18 +58,31 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                handleQuery={handleQuery}
+                query={query}
+                sortBy={sortBy}
+                handleSortType={handleSortType}
```

Comment:
Follow [these](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) naming conventions for methods everywhere